addons/website_blog/static/src/js/website_blog.js

```diff
@@ -17,6 +17,15 @@ publicWidget.registry.websiteBlog = publicWidget.Widget.extend({
      */
     start: function () {
         $('.js_tweet, .js_comment').share({});
+        // TODO in master: do the following fix in XML instead.
+        const anchorEl =
+            this.$target[0].querySelector('#o_wblog_blog_top a[href^="/blog?blog=blog.blog"]');
+        if (anchorEl) {
+            const blogPostUrl = this.$target[0]
+                .querySelector('#o_wblog_blog_top [data-blog-id]').getAttribute('href');
+            const pathToPost = blogPostUrl.split('/');
+            anchorEl.href = `/${pathToPost[1]}/${pathToPost[2]}/`;
+        }
```

Comment:
What would be the XML fix ? :thinking: Is not this about fixing the blog_url used to compute that href? :thinking:

Reply:
Indeed I was initially thinking to replace `t-attf-href="#{blog_url(blog=first_post.blog_id)}"` by `t-attf-href="/blog/#{slug(first_post.blog_id)}"` but you are right, it's better to fix `blog_url` in python